### DISCUSSION TITLE
Graph Functionality

### DISCUSSION THREAD

**Discussion by cloud9-kj** [google/timesketch#1815] [Q&A, quick solutions, support] [answered]
Hi, 

Is there a way to add the functionality? A step-by-step or some sort of documentation would be really helpful. Thanks
I found a URL but it no longer exists.

**Comment by kiddinn** [comment, marked as answer]
We should probably create an issue to track that.. did that already, see (here)[https://github.com/google/timesketch/issues/1819].

But for now you can take a look at the current plugins that are stored (here)[https://github.com/google/timesketch/tree/master/timesketch/lib/graphs] and try to copy and modify them

**Reply by cloud9-kj** [reply]
Is it possible to see the list of tags available in TimeSketch?

**Reply by berggren** [reply]
A view of what tags that is part of an investigation is WIP but will soon be available:
Part of: https://github.com/google/timesketch/pull/1838
